### ISSUE TITLE
Implement search page title fix

### ISSUE THREAD

**Issue by nichgray** [CDRH/earlywashingtondc#256]
Ideally we should get the search term into the page title. Review Orchid changes and see if they can be ported over.

**Comment by nichgray** [comment]
This should be resolved with https://github.com/CDRH/earlywashingtondc/pull/251/changes/cb8f20348174e14fb9b82ce185d28f764476a851.
